easybuild/easyconfigs/m/Mosek/Mosek-9.3.20-foss-2021b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'Mosek'
+version = '9.3.20'
+
+homepage = 'http://www.mosek.com/'
+description = """ MOSEK is a package for large-scale convex and mixed-integer optimization, including LP, QP, SOCP, 
+exponential and power cone problems, SDP, QCQP and MIP.
+Documentation and examples can be found at the MOSEK website
+MOSEK requires a license. Free of charge trial licenses as well as licenses for non-commercial academic use can be
+obtained from the MOSEK website."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+sources = ['%(name)s-%(version)s-cp39-cp39-manylinux2014_x86_64.whl']
+checksums = ['508b8b1fad9e19ae23e9bc94070a31199d975efc52301de79bd5300576f86e31']
+
+dependencies = [
+    ('Python', '3.9.6'),
```

Comment:
@ysagon looks like you need to add `SciPy-bundle` because this package needs `bumpy`.
Here's the error I saw during a test build:
```
== 2022-07-26 11:35:26,243 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): `pip check` failed:
mosek 9.3.20 requires numpy, which is not installed.
```

Reply:
@verdurin done